src/components/services/Services.vue

```diff
@@ -241,8 +262,8 @@ export default {
       );
       this.displayAddServiceForm = false;
     },
-    removeAccents(text){
-      return text.normalize('NFD').replace(/[\u0300-\u036f]/g, "");
+    removeAccents(text) {
+      return text.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Comment:
single quotes are recommended in java for string, and more often used. I suggest using single quotes instead of double quotes

Reply:
Da mie asa imi face ide-ul automat , desi e pun de alea simple. :)
Folosesc visual studio code